internal/builtinfunctions/functions.go

```diff
@@ -79,27 +80,33 @@ func getFloatToIntFunction() schema.CallableFunction {
 		"floatToInt",
 		[]schema.Type{schema.NewFloatSchema(nil, nil, nil)},
 		schema.NewIntSchema(nil, nil, nil),
-		false,
+		true,
 		schema.NewDisplayValue(
 			schema.PointerTo("floatToInt"),
 			schema.PointerTo(
-				"Converts a float type into an integer type by discarding the fraction."+
+				"Returns an integer representation of the inputted floating-point number.\n"+
```

Comment:
Again, try "Returns an integer representation of the floating-point parameter" (or "value"), without the "inputting". 😆

Reply:
Changed to use parameter.